docs/markdown/ios/card/card.pins.md

```diff
@@ -1,29 +1,107 @@
+
+<TwoCol>
+<Group>
+<Do title="When to use"/>
+- To recommend a single landing experience containing related Pins (e.g. topical Pin feed, more ideas for your board) to a user based on their engagement, interests, boards, demographic information or other signals
+
+</Group>
+<Group>
+<Dont title="When not to use" />
+- To suggest Pins that are not related or within a defined grouping
+- When recommending more than one landing experience (e.g. 2 or more topical Pin feeds)
+- When the goal is to direct Pinners to a specific place in a destination (e.g. a slot position within a stream).
+</Group>
+</TwoCol>
+
+### Best practices
+
+<TwoCol>
+<Group>
+<Do title="Do"/>
+- All images should be equal in size, whether 2up, 3up or 4up
+- The entire Card.Pins preview block is a single tap target that directs Pinners to a single destination
+
+</Group>
+<Group>
+<Dont title="Don't" />
+- Make each image its own tap target. The entire Card.Pins preview block should be a single tap target.
+</Group>
+</TwoCol>
+
+For general specifications, refer to the [MontageGrid & MontageRow documentation](https://www.figma.com/file/AHcKJDgb7E7YswlgW1wY8E/Gestalt-for-iOS?node-id=19102%3A71831&t=uAttTYuo18hxKqUw-1).
+
+### Accessibility
+
+People use Apple’s accessibility features, such as reduced transparency, VoiceOver, and increased text size, to personalize how they interact with their devices. Supporting these personalizations ensures that everyone has a great user experience. See Apple’s Human Interface Guidelines and documentation about accessibility for iOS:
+
+[Accessible design on iOS](https://developer.apple.com/design/human-interface-guidelines/accessibility/overview/introduction/)
+[Accessible development on iOS](https://developer.apple.com/accessibility/ios/)
+
+### Design tokens
+
+<iframe style={{border:0}} width="100%" height="178" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FAHcKJDgb7E7YswlgW1wY8E%2FGestalt-for-iOS%3Ftype%3Ddesign%26node-id%3D33462%253A75994%26mode%3Ddesign%26t%3DIclHzfbakvSzsqp8-1" allowFullScreen></iframe>
+
+### Anatomy
+
+<iframe style={{border:0}} width="100%" height="450" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FAHcKJDgb7E7YswlgW1wY8E%2FGestalt-for-iOS%3Ftype%3Ddesign%26node-id%3D19800%253A76473%26mode%3Ddesign%26t%3DmAYek1WBSRRnRvq3-1" allowFullScreen></iframe>
+
+### Variants
+
+#### 2-up
+
+<iframe style={{border:0}} width="100%" height="312" src="https://www.figma.com/file/AHcKJDgb7E7YswlgW1wY8E/Gestalt-for-iOS?type=design&node-id=19826%3A75261&mode=design&t=IclHzfbakvSzsqp8-1" allowFullScreen></iframe>
+
+#### 3-up (default)
+
+<iframe style={{border:0}} width="100%" height="312" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FAHcKJDgb7E7YswlgW1wY8E%2FGestalt-for-iOS%3Ftype%3Ddesign%26node-id%3D19800%253A76866%26mode%3Ddesign%26t%3DIclHzfbakvSzsqp8-1" allowFullScreen></iframe>
+
+#### 4-up
```

Comment:
Sorry, wrong embed link. use this one: https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FAHcKJDgb7E7YswlgW1wY8E%2FGestalt-for-iOS%3Ftype%3Ddesign%26node-id%3D19826%253A75351%26mode%3Ddesign%26t%3DmAYek1WBSRRnRvq3-1

height: 360